ansible_image_validation/validation-playbooks/per-vm-validation.yaml

```diff
@@ -88,4 +88,39 @@
     line: "\n NVME validation failed since pci driver is not present. "
     create: yes
     state: present 
-  when: ("pci" not in check_pci_allimages.stdout)         
\ No newline at end of file
+  when: ("pci" not in check_pci_allimages.stdout)
+
+- name: Check for Rhui client package in the image
+  when: license_type != 'byos'
+  block:
+  - name: Gather the package facts
+    ansible.builtin.package_facts:
+      manager: auto
+
+  - name: Check for rhui package details
+    set_fact:
+      rhui_package: "{{ ansible_facts.packages | dict2items | selectattr('key', 'match', 'rhui-azure-rhel') }}"
+
+  - name: Check for rhui package count
```

Comment:
Please add the validation to check that the 'correct' client package is embedded in the image also? Eg: Base image does not include 'sap' client package scenario.

Reply:
yes, I thought about it. Have a plan for this but this is not in the scope of this PR